### ISSUE TITLE
Hide native checkbox and radio in a more accessible way

### ISSUE THREAD

**Issue by drazik** [cozy/cozy-ui#1122]
Today we use `display: none` and `visibility: hidden` to hide checkboxes and radio buttons. This is not a good way to do this as these controls will be totally inaccessible for screen readers users.

It also make testing of checkbox and radio buttons hard in testcafé. It throws the following error: `The element that matches the specified selector is not visible`.

See https://webaim.org/techniques/css/invisiblecontent/ for techniques to hide content will staying accessible.

**Comment by ptbrowne** [comment]
With this, the input is selectable and focusable via tab + space, while being 

```
input {
  opacity: 0.00;
  width: 1px;
  height: 1px;
}
```

**Comment by ptbrowne** [comment]
Would you be up to take this issue drazik ?

**Comment by ptbrowne** [comment]
Ping au cas ou :)

**Comment by drazik** [comment]
Yep, assigning myself :)
